File: riju/School Practical/1.py
def sumrec(list1,num):
    if num ==  0:
        return 0
    else:
        return int(list1[num-1])+ sumrec(list1,num-1)

emp=[]

a = 0
while 2>=1:
    a += 1
    f =input("Enter a number or press enter to terminate ")
    if str(f) == "":
        break
    else:
        emp.append(f)

print("The sum of all entities is = ",sumrec(emp,len(emp)))

File: SDF-3/1.c
#include <stdio.h>
int main()
{

    int x, i, j;
    printf("Enter the number of rows\n");
    scanf("%d", &x);

    for (j = 0; j < x; j++)
    {
        printf("*");
    }
    printf("\n");
    for (i = (x - 1); i > 0; i--)
    {
        for (j = 0; j < (x - i); j++)
        {
            printf(" ");
        }
        for (j = i; j > 0; j--)
        {
            if (j < i && j > 1)
            {
                printf(" ");
            }
            else
            {
                printf("*");
            }
        }
        printf("\n");
    }
    return 0;
}
